src/components/ha-form/ha-form.ts

```diff
@@ -92,6 +92,7 @@ export class HaForm extends LitElement implements HaFormElement {
                   schema: item,
                   data: getValue(this.data, item),
                   label: this._computeLabel(item, this.data),
+                  helper: this._computeHelper(item),
                   disabled: this.disabled,
                   hass: this.hass,
                   computeLabel: this.computeLabel,
```

Comment:
Why do we pass these functions along?

Reply:
I didn't touch that code.  I believe they are needed by the grid schema.

Reply:
oh right